roles/hadoop/templates/hadoop-hdfs.systemd.j2

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=hadoop-hdfs
+After=syslog.target network.target
+
+[Service]
+User=ubuntu
+Type=forking
+Environment="JAVA_HOME=/usr/lib/jvm/java-8-oracle"
```

Comment:
Fixed in:
https://github.com/nlesc-sherlock/emma/commit/b77a04dc06bc76b725007dc1ea0482bbce02cc60
and:
https://github.com/nlesc-sherlock/emma/commit/854de7b8005299d9875a2999bd6e9838d9c71982